arm_robots/scripts/cartesian_teleop.py

```diff
@@ -21,8 +22,12 @@ class CartesianTeleop:
     The "x" button (on logitech controller) will switch which tool name is being controller
 
     """
-    def __init__(self, robot_name: str, group_name: str, tool_names: List[str]):
-        self.gamepad = Logitech()
+    def __init__(self, robot_name: str, group_name: str, tool_names: List[str], gamepad_name: str):
+        if gamepad_name == 'logitech':
+            self.gamepad = Logitech()
+        else:
```

Comment:
elif and `raise`. not sure what other controllers will be in the lab in the future